CHANGELOG.md

```diff
@@ -1,6 +1,70 @@
-### citus v7.0.0 (unreleased) ###
+### citus v7.0.0 (August 28, 2017) ###
 
-* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
+* Adds support for PostgreSQL 10
+
+* Drops support for PostgreSQL 9.5
+
+* Adds support for multi-row `INSERT`
+
+* Adds support for router `UPDATE` and `DELETE` queries with subqueries
+
+* Adds infrastructure to distributed deadlock detection
+
+* Adds maintenance daemon for distributed deadlock detection and 2PC recovery
```

Comment:
I think we don't do 2PC recovery automatically yet. If that is the case, I think we can drop this line.